lib/new_relic/agent/instrumentation/roda/chain.rb

```diff
@@ -0,0 +1,47 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+# rubocop:disable Lint/DuplicateMethods
```

Comment:
Rubocop was upset about defining duplicate methods. I'm not sure why we are getting yelled at this time but seems to pass in other chain instrumentations.